src/main/java/com/epam/newsPortal/controller/ArticleController.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.newsPortal.controller;
+
+import com.epam.newsPortal.entity.Article;
+import com.epam.newsPortal.entity.Category;
+import com.epam.newsPortal.entity.Editor;
+import com.epam.newsPortal.service.ArticleService;
+import com.epam.newsPortal.service.CategoryService;
+import com.epam.newsPortal.service.EditorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+
+@Controller
+@RequestMapping(value = "/article", produces = "text/plain;charset=UTF-8")
+
+public class ArticleController {
+
+    private ArticleService articleService;
+    private CategoryService categoryService;
+    private EditorService editorService;
+
+
+    @Autowired
+    public ArticleController(ArticleService articleService, CategoryService categoryService, EditorService editorService) {
+        this.articleService = articleService;
+        this.categoryService = categoryService;
+        this.editorService = editorService;
+
+    }
+    @GetMapping("/addArticle")
+    public String addArticle(@ModelAttribute("article")Article article,@RequestParam("editorId")Long editorId, Model model){
+        model.addAttribute("categoryList", categoryService.getAll());
+        Editor editor = editorService.getEditor(editorId);
+        System.out.println("Editor from /addArticle "+ editor);
+        model.addAttribute("editor", editor);
```

Comment:
editor is very strange think/
Be ready to understand what is it and why do you use in in ALL methods